src/commands/mobile_app/upload.rs

```diff
@@ -67,12 +102,24 @@ pub fn execute(matches: &ArgMatches) -> Result<()> {
         .get_many::<String>("paths")
         .expect("paths argument is required");
 
-    let sha = matches
-        .get_one("sha")
+    let head_sha = matches
+        .get_one("head_sha")
         .map(String::as_str)
         .map(Cow::Borrowed)
         .or_else(|| vcs::find_head().ok().map(Cow::Owned));
 
+    // TODO: Implement default values
+    let base_sha = matches.get_one("base_sha").map(String::as_str);
+    let vcs_provider = matches.get_one("vcs_provider").map(String::as_str);
+    let head_repo_name = matches.get_one("head_repo_name").map(String::as_str);
+    let base_repo_name = matches.get_one("base_repo_name").map(String::as_str);
+    let head_ref = matches.get_one("head_ref").map(String::as_str);
+    let base_ref = matches.get_one("base_ref").map(String::as_str);
+    let pr_number = matches
+        .get_one("pr_number")
+        .map(String::as_str)
+        .and_then(|s| s.parse::<i32>().ok());
```

Comment:
Once you change the `pr_number`'s type to `u32` in the struct, and once you add the `value_parser` above, you will be able to simplify this line greatly:

```suggestion
    let pr_number = matches.get_one("pr_number");
```